src/App.js

```diff
@@ -1,26 +1,16 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import Fetchitems from "./component/Fetchitems"
+import AddItem from "./component/AddItem";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
Random `<div>` tags in the HTML can reduce the accessibility of a page or app. A better tag to use here might be `<main>` to denote to screen reader software that its contents make up the primary content of the page. Another possible approach for avoiding an extra tag altogether would be to use a `Fragment` component, which is one that is used to group items without actually rendering any HTML. You can read about them here: https://reactjs.org/docs/fragments.html ... In this case, I'd _probably_ use a `Fragment` because `Fetchitems` and `AddItem` appear to be their own pages. But I'll leave it up to you which approach you take!

Reply:
Fragment it is!